### ISSUE TITLE
wikiqid Apple gets an exception (node 5.7.1)

### ISSUE THREAD

**Issue by Hunter-Github** [maxlath/wikidata-cli#1]
wikidata-cli 1.3.1

```
$ wikiqid Apple

TypeError: Cannot convert undefined or null to object
    at Function.keys (native)
    at parseIds (~/.nodebrew/node/v5.7.1/lib/node_modules/wikidata-cli/bin/wikiqid:43:20)
    at tryCatcher (~/.nodebrew/node/v5.7.1/lib/node_modules/wikidata-cli/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (~/.nodebrew/node/v5.7.1/lib/node_modules/wikidata-cli/node_modules/bluebird/js/release/promise.js:502:31)
    at Promise._settlePromise (~/.nodebrew/node/v5.7.1/lib/node_modules/wikidata-cli/node_modules/bluebird/js/release/promise.js:559:18)
    at Promise._settlePromise0 (~/.nodebrew/node/v5.7.1/lib/node_modules/wikidata-cli/node_modules/bluebird/js/release/promise.js:604:10)
    at Promise._settlePromises (~/.nodebrew/node/v5.7.1/lib/node_modules/wikidata-cli/node_modules/bluebird/js/release/promise.js:683:18)
    at Async._drainQueue (~/.nodebrew/node/v5.7.1/lib/node_modules/wikidata-cli/node_modules/bluebird/js/release/async.js:138:16)
    at Async._drainQueues (~/.nodebrew/node/v5.7.1/lib/node_modules/wikidata-cli/node_modules/bluebird/js/release/async.js:148:10)
    at Immediate.Async.drainQueues [as _onImmediate] (~/.nodebrew/node/v5.7.1/lib/node_modules/wikidata-cli/node_modules/bluebird/js/release/async.js:17:14)
    at processImmediate [as _immediateCallback] (timers.js:383:17)
```

**Comment by maxlath** [comment]
it seems that the server didn't returned the expected response, I made a patch to get better error reports, could you try to update to the latest version and retry?

**Comment by Hunter-Github** [comment]
Gladly. The root of the problem:

```
Context:
 { servedby: 'mw1277',
  warnings: { wbgetentities: { '*': 'Unrecognized value for parameter \'sites\': uswiki' } },
  error: 
   { code: 'param-missing',
     info: 'A parameter that is required was missing. (Either provide the item "ids" or pairs of "sites" and "titles" for corresponding pages)',
     messages: [ [Object] ],
     '*': 'See https://www.wikidata.org/w/api.php for API usage' } }
```

**Comment by maxlath** [comment]
oh, seems that the local language detection function was just plain wrong, returning the region instead of the language two letters code (in your case `us` instead of `en`, thus the wrong site parameter `uswiki`). I fixed it there bca05bb and published it as a new patch. Could you confirm that it fixes the issue?

**Comment by Hunter-Github** [comment]
It finds New Delhi and Apple (the fruit).
